azkaban-web-server/build.gradle

```diff
@@ -50,27 +71,55 @@ sourceSets {
   }
 }
 
-bower {
-  installBase = 'bower' // <1>
+task movingJsTojsToPackage(dependsOn: ['npm_install']) {
 
-  'moment'('2.14.1'){
-    source 'min/moment.min.js' >> '/'
+  doLast {
+    copy {
+      from "node_modules/later/later.min.js"
+      into 'build/jsToPackage'
+    }
+    copy {
+      from "node_modules/moment/min/moment.min.js"
+      into 'build/jsToPackage'
+    }
+    copy {
+      from "node_modules/eonasdan-bootstrap-datetimepicker/build/js/bootstrap-datetimepicker.min.js"
+      into 'build/jsToPackage'
+    }
+    copy {
+      from "node_modules/moment-timezone/builds/moment-timezone-with-data-2010-2020.min.js"
+      into 'build/jsToPackage'
+    }
   }
+}
 
-  'moment-timezone'('0.5.5') {
-    source 'builds/moment-timezone-with-data-2010-2020.min.js'  >> '/'
-  }
+task cleanAll {
+  delete '.gradle'
```

Comment:
You don't need to delete hidden directories like .gradle. They are used by gradle to manage different gradle versions and dependencies.